hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -530,7 +525,11 @@ public void onCompactionCompleted(RocksDB db,
               compactionLogEntry.getOutputFileInfoList(),
               compactionLogEntry.getDbSequenceNumber());
           for (String inputFile : inputFileCompactions.keySet()) {
-            inflightCompactions.remove(inputFile);
+            CompactionFileInfo removed = inflightCompactions.remove(inputFile);
+            if (removed == null) {
+              LOG.info("Input file not found in inflightCompactionsMap : {} which should have been added on " +
+                      "compactionBeginListener.", inputFile);
```

Comment:
What is `compactionBeginListener` ?  Could you show the code/doc?

Please include the job id and column family name in the message.

Reply:
When you say show code/doc do you mean I should add a comment in the code here?

Reply:
done added some comments